src/bin/vip-import-sql.js

```diff
@@ -265,6 +265,15 @@ If you are confident the file does not contain unsupported statements, you can r
 	return getTableNames();
 }
 
+function validateFilename( filename ) {
+	const re = /^[a-z0-9!\-_\.\*\'()]+$/i;
+
+	// Exits if filename contains anything outside a-z A-Z ! - _ . * ( )
```

Comment:
`!` and `*` are allowed here but not mentioned on the [public docs](https://docs.wpvip.com/how-tos/migrate-content-databases/#h-limitations)

Reply:
Ahh nice catch.

Reply:
https://github.com/Automattic/vip-cli/pull/1217/commits/14fbb61cb64fb46bbc17855867330155987c63ae